operations/mimir/consul.libsonnet

```diff
@@ -2,11 +2,12 @@ local consul = import 'consul/consul.libsonnet';
 
 {
   _config+:: {
-    consul_replicas: 1,
+    consul_enabled: true,
+    consul_replicas: if !$._config.consul_enabled then 0 else 1,
```

Comment:
`consul_replicas` is only used when `consul_enabled` is true, so we don't need to modify its value.